CMakeLists.txt

```diff
@@ -61,6 +61,10 @@ endif()
 # PIC (Position Independent Code) ensures .a files can be linked to executables that have PIE enabled
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+if (DO_DISABLE_EXCEPTIONS)
+    #TODO(jimson): Set cmake flags for disabling exceptions and rtti, there is no cmake setting for this, so we must set the compiler-specific flag
+endif()
```

Comment:
This flag is compiler dependent - not sure if we should explicitly set it here (might help us actually verify our code will compile against -fno-except) but also we dont have a way to know if this is true for all compiler/toolchain combos we need.

Reply:
We could set the correct flag based on the compiler but I agree this isn't blocking.

Reply:
Yes, I also need to modify the comment because it reads a little confusingly. I'll modify it to say 
"There is no cmake option for disabling exceptions/RTTI, so we must set a compiler-specific flag if we want to enforce disabling exceptions"